CHANGELOG.asciidoc

```diff
@@ -0,0 +1,501 @@
+ifdef::env-github[]
+NOTE: Release notes are best read in our documentation at
+https://www.elastic.co/guide/en/apm/agent/go/current/release-notes.html[elastic.co]
```

Comment:
This is referencing the go agent, it should be `https://www.elastic.co/guide/en/apm/agent/rum-js/current/release-notes.html`